src/DssSpell.sol

```diff
@@ -18,77 +18,28 @@ pragma solidity 0.8.16;
-import "dss-interfaces/dss/GemJoinAbstract.sol";
-import "dss-interfaces/dss/MedianAbstract.sol";
-import "dss-interfaces/ERC/GemAbstract.sol";
-
-interface Initializable {
-    function init(bytes32 ilk) external;
-}
 
 interface VatLike {
+    function Line() external view returns (uint256);
+    function file(bytes32 what, uint256 data) external;
     function ilks(bytes32 ilk) external view returns (uint256 Art, uint256 rate, uint256 spot, uint256 line, uint256 dust);
 }
 
-interface RwaLiquidationLike {
-    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
-    function init(bytes32 ilk, uint256 val, string calldata doc, uint48 tau) external;
-}
-
-interface RwaUrnLike {
-    function lock(uint256 wad) external;
-    function hope(address usr) external;
+interface DssVestLike {
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function file(bytes32, uint256) external;
+    function restrict(uint256) external;
 }
-
-interface RwaInputConduitLike {
-    function mate(address usr) external;
-    function file(bytes32 what, address data) external;
-}
-
-interface RwaOutputConduitLike {
-    function file(bytes32 what, address data) external;
-    function hope(address usr) external;
-    function mate(address usr) external;
-    function kiss(address who) external;
-}
-
-interface PoolConfiguratorLike {
-    struct InitReserveInput {
-        address aTokenImpl;
-        address stableDebtTokenImpl;
-        address variableDebtTokenImpl;
-        uint8 underlyingAssetDecimals;
-        address interestRateStrategyAddress;
-        address underlyingAsset;
-        address treasury;
-        address incentivesController;
-        string aTokenName;
-        string aTokenSymbol;
-        string variableDebtTokenName;
-        string variableDebtTokenSymbol;
-        string stableDebtTokenName;
-        string stableDebtTokenSymbol;
-        bytes params;
-    }
-    function initReserves(InitReserveInput[] calldata input) external;
-    function configureReserveAsCollateral(
-        address asset,
-        uint256 ltv,
-        uint256 liquidationThreshold,
-        uint256 liquidationBonus
-    ) external;
-    function setBorrowableInIsolation(address asset, bool borrowable) external;
-    function setDebtCeiling(address asset, uint256 newDebtCeiling) external;
-}
-
-interface AaveOracleLike {
-    function setAssetSources(address[] calldata assets, address[] calldata sources) external;
+interface RwaLiquidationLike {
+    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
+    VatLike  internal immutable vat  = VatLike(DssExecLib.vat());
```

Comment:
This is inconsistent with line 69, please use one or another way to init deps

Reply:
i assume this is resolved.